packages/core/components/FileDetails/FileDetails.module.css

```diff
@@ -93,6 +108,8 @@
 
 .file-thumbnail-container {
     margin: var(--margin) 10px 10px 10px; /* defined in App.module.css under .root{} */
+    margin-left: auto;
+    margin-right: auto;
```

Comment:
Would be good to clean up these margin declarations: the first one is declaring for all dimensions, the second two are overriding the first.